app/views/partials/foyer/ressources/montants.html

```diff
@@ -92,6 +156,50 @@ <h3 class="panel-title"><span class="short-label">{{ individuVM.label }}</span>
           </div>
         </div>
       </div>
+      <div class="form-group">
+        <div class="checkbox">
+          <label>
+            <input
+              type="checkbox"
+              ng-model="ressource.employes">
+              J'ai au moins un employé
+          </label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+    <div class="panel panel-montant tns"
+   ng-if="ressource.type.id === 'exploitantAgricole'">
```

Comment:
Either keep the current heavyweight yoda-style + triple-equal, or switch to a simpler double-equal, var-first style.